tests/integration/static_ip_address_test.py

```diff
@@ -537,3 +537,40 @@ def test_add_iface_with_same_static_ipv6_address_to_existing(
     libnmstate.apply(desired_state)
 
     assertlib.assert_state(desired_state)
+
+
+@pytest.mark.xfail(reason='https://bugzilla.redhat.com/1760800', strict=True)
+def test_add_iface_with_static_ipv6_expanded_format(eth1_up):
+    ipv6_addr_lead_zeroes = '2001:0db8:85a3:0000:0000:8a2e:0370:7331'
+    desired_state = {
+        INTERFACES: [
```

Comment:
I can compromise and at least move us a bit in the right direction by not adding more keys that will need to be updated :)

Reply:
> We no longer use this constant, but it will be unfair to ask to use the alternative now.
> There are also usages of non constant for the interface keys (`name`, `type`, etc...).
> 
> @ffmancera , I hope you can help to change this throughout the project.

Yes, I've sent a PR #549.